src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -117,3 +137,119 @@ export const privateMedicalRecordsIntro = ({ formData }) => {
+  }
+
+  validatePostalCodes(errors, formData);
+}
+
+export const recordReleaseWarning = () => {
+  return (
+    <div className="usa-alert usa-alert-warning no-background-image">
+      <span>Limiting consent means that your doctor can only share records that are directly related to your condition. This could add to the time it takes to get your private medical records.</span>
+    </div>
+  );
+};
+
+export const documentDescription = () => {
+  return (
+    <div>
+      <p>File upload guidelines:</p>
+      <ul>
+        <li>File types you can upload: .pdf, .jpeg, .gif, .tiff, or .png</li>
+        <li>Maximum file size: 25 MB</li>
+      </ul>
+      <p><em>Large files can be more difficult to upload with a slow Internet connection</em></p>
+    </div>
+  );
+};
+
+export const additionalDocumentDescription = () => {
+  return (
+    <div>
+      <p>You mentioned that you have other evidence, like lay statements that you’d like to add to support your claim. You can upload them here.</p>
+      <p>File upload guidelines:</p>
+      <ul>
+        <li>File types you can upload: .pdf, .jpeg, .gif, .tiff, or .png</li>
+        <li>Maximum file size: 25 MB</li>
+      </ul>
+      <p><em>Large files can be more difficult to upload with a slow Internet connection</em></p>
+    </div>
+  );
+};
+
+const documentLabels = {
+  1: 'Discharge',
+  2: 'Marriage related',
+  3: 'Dependent related',
+  // 4: 'VA preneed form',
+  5: 'Letter',
+  6: 'Other'
+};
+
+export const evidenceSummaryView = ({ formData }) => {
+  // console.log(formData, pagePerItemIndex);
+  // TODO: need to make additional documents same so can use type below
+  const { treatments: VATreatments, privateRecordReleases, privateRecords, additionalDocuments } = formData;
+  // TODO: update bullet style for additional documents
+  return (
+    <div>
+      <ul>
+        {VATreatments &&
+        <li>We will get your medical records from {VATreatments.map((VAcenter, idx) => {
+          const notLast = idx < (VATreatments.length - 1);
+          return <strong key={idx}>{`${VAcenter.treatment.treatmentCenterName}${notLast ? ', ' : ''}`}</strong>;
+        })}.</li>}
+        {privateRecordReleases &&
+        <li>We will get your private medical records from {privateRecordReleases.map((release, idx) => {
+          const notLast = idx < (privateRecordReleases.length - 1);
+          return <strong key={idx}>{`${release.privateRecordRelease.treatmentCenterName}${notLast ? ', ' : ''}`}</strong>;
+        })}.</li>}
+        {privateRecords && <li>We have the private medical records you uploaded.</li>}
+        {additionalDocuments &&
+        <li>We have this additional evidence that you uploaded:
+          <ul>
+            {additionalDocuments.map((document, id) => {
+              return (<li className="dashed" key={id}>
+                <strong>{`${documentLabels[document.attachmentId]} (${document.name})`}</strong>
```

Comment:
@goldenmeanie we display the document name here as well as its type, in case the applicant provides multiple documents of the same type. Is this a clear way to convey what's been uploaded to the applicant (the design doesn't include the file name)?